column.go

```diff
@@ -35,6 +35,7 @@ func (l *BufferLen) Bind(h api.SQLHSTMT, idx int, ctype api.SQLSMALLINT, buf []b
 // Column provides access to row columns.
```

Comment:
Add

```
Fixes #174
```

to the end of your commit message. So future code readers can determine why we made that change.